java-manta-client/src/test/java/com/joyent/manta/client/crypto/MantaEncryptedObjectInputStreamTest.java

```diff
@@ -932,9 +931,6 @@ private void canReadByteRange(SupportedCipherDetails cipherDetails,
              * the binary of the actual object (even if the range went beyond). */
             long ciphertextByteRangeLength = Math.min(ciphertextRangeMaxBytes, ciphertextRangeRequestBytes);
 
-            if (unboundedEnd)
-                ciphertextByteRangeLength = ciphertextSize - ciphertextStart; // include MAC
```

Comment:
That's kind of part of my point; the test has additional logic that either overlaps with other classes or is missing.

I remember reading `ENCRYPTION_PLAINTEXT_CONTENT_LENGTH` in the original specification; reading through the code though, the impression I got was that it wasn't a value that could be *guaranteed* to be accurate.

If you check the branch the byte-range pull request was based on, the implementation I chose was to depend on rfc values, since the *manta* server should be able to compute object length atomically, and plaintext/ciphertext boundaries are pure functions of the ciphers and object length.  `Content-Range` object length is not *required* per rfc-7233... although worst case fallback could still be making an additional `head` request to *manta*.